services/pom.xml

```diff
@@ -34,6 +34,16 @@
         <module>sts</module>
         <module>wsn</module>
         <module>ws-discovery</module>
-        <module>xkms</module>
```

Comment:
@reta It's only xkms/itests that needs to be excluded, not the whole xkms module.

Reply:
👍 oops, sorry, fixed